docs/guides/export.md

```diff
@@ -29,4 +29,61 @@ Once your export resource is configured, run `terraform init` to set up Terrafor
 
 If state is exported, the config file may not be able to be applied to another org as it likely contains ID references to objects in the current org. If you choose not to export the state file, the standalone `.tf.json` or `.tf` config file will be stripped of all reference attribute values that cannot be mapped to exported resources. For example if you only export users, any attributes that reference other object types (roles, skills, etc.) will be removed from the config. This is necessary as it would not be possible to apply configuration with references to IDs from a different org.
 
-If exported resources contain references to objects that we don't intend to manage with Terraform or if they cannot be resolved using an API call then a variable will be generated to refer to that object. A definition for that variable will be provided in a generated `terraform.tfvars` file. The reference variables must be filled out with the values of the corresponding resources in a different org before being applied to it.
\ No newline at end of file
+If exported resources contain references to objects that we don't intend to manage with Terraform or if they cannot be resolved using an API call then a variable will be generated to refer to that object. A definition for that variable will be provided in a generated `terraform.tfvars` file. The reference variables must be filled out with the values of the corresponding resources in a different org before being applied to it.
+
+# Filtering Resources with Regular Expressions
+
+In your Terraform setup, regular expressions can be employed to selectively include or exclude certain resources. Here’s a concise way to do it:
+
+## Include Filter:
+
+If you want to include resources that begin or end with “dev” or “test”, use the following format:
+
+
+```hcl
+resource "genesyscloud_tf_export" "include-filter" {
+  directory = "./genesyscloud/include-filter"
+  export_as_hcl = true
+  log_permission_errors = true
+  include_filter_resources = ["genesyscloud_group::.*(?:dev|test)$"]
```

Comment:
Could you line up all the equals characters vertically? The resource is totally valid the way you have it, but it's much tidier looking when they're lined up. Same with the other resource below